frontend/src/tests/pages/SearchByInstructor/SearchByInstructorPage.test.js

```diff
@@ -0,0 +1,91 @@
+// import { render, screen, waitFor } from "@testing-library/react";
+import { render} from "@testing-library/react";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+import axios from "axios";
+import AxiosMockAdapter from "axios-mock-adapter";
+// import userEvent from "@testing-library/user-event";
+// import { allTheSubjects } from "fixtures/subjectFixtures";
+// import { oneSection } from "fixtures/sectionFixtures";
+import { apiCurrentUserFixtures } from "fixtures/currentUserFixtures";
+import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+import SearchByInstructorPage from 'main/pages/SearchByInstructor/SearchByInstructorPage';
+
+const mockToast = jest.fn();
+jest.mock("react-toastify", () => {
+  const originalModule = jest.requireActual("react-toastify");
+  return {
+    __esModule: true,
+    ...originalModule,
+    toast: (x) => mockToast(x),
+  };
+});
+
+describe("Search by Instructor Page tests", () => {
+  const axiosMock = new AxiosMockAdapter(axios);
+
+  beforeEach(() => {
+    axiosMock.resetHistory();
+    axiosMock
+      .onGet("/api/currentUser")
+      .reply(200, apiCurrentUserFixtures.userOnly);
+    axiosMock
+      .onGet("/api/systemInfo")
+      .reply(200, systemInfoFixtures.showingNeither);
+  });
+
+  const queryClient = new QueryClient();
+  test("renders without crashing", () => {
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <SearchByInstructorPage />
+        </MemoryRouter>
+      </QueryClientProvider>
+    );
+  });
+
+//   test("calls UCSB section search api correctly with 1 section response", async () => {
+//     axiosMock.onGet("/api/UCSBSubjects/all").reply(200, allTheSubjects);
+//     axiosMock
+//       .onGet("/api/sections/basicsearch")
+//       .reply(200, oneSection);
+
+//     render(
+//       <QueryClientProvider client={queryClient}>
+//         <MemoryRouter>
+//           <SectionSearchesIndexPage />
+//         </MemoryRouter>
+//       </QueryClientProvider>
+//     );
+
+//     const selectQuarter = screen.getByLabelText("Quarter");
+//     userEvent.selectOptions(selectQuarter, "20222");
+//     const selectSubject = screen.getByLabelText("Subject Area");
```

Comment:
Commented out tests shouldn't go into a PR intended for merging into main.

Could you remove this test?